[2023-10-17T11:19:12.291+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [queued]>
[2023-10-17T11:19:12.303+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [queued]>
[2023-10-17T11:19:12.303+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:12.334+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-15 00:00:00+00:00
[2023-10-17T11:19:12.342+0000] {standard_task_runner.py:57} INFO - Started process 573 to run task
[2023-10-17T11:19:12.345+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-15T00:00:00+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpq3vblm1c']
[2023-10-17T11:19:12.347+0000] {standard_task_runner.py:85} INFO - Job 78: Subtask waiting_for_data
[2023-10-17T11:19:12.403+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:12.486+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-15T00:00:00+00:00'
[2023-10-17T11:19:12.494+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:12.494+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:12.494+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:12.495+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:12.504+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210315T000000, start_date=20231017T111912, end_date=20231017T111912
[2023-10-17T11:19:12.565+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:12.618+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:41.255+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [queued]>
[2023-10-18T10:42:41.265+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [queued]>
[2023-10-18T10:42:41.265+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:41.277+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-15 00:00:00+00:00
[2023-10-18T10:42:41.285+0000] {standard_task_runner.py:57} INFO - Started process 1536 to run task
[2023-10-18T10:42:41.288+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-15T00:00:00+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmptbr8y8r4']
[2023-10-18T10:42:41.289+0000] {standard_task_runner.py:85} INFO - Job 77: Subtask waiting_for_data
[2023-10-18T10:42:41.343+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:41.423+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-15T00:00:00+00:00'
[2023-10-18T10:42:41.451+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:41.456+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210315T000000, start_date=20231018T104241, end_date=20231018T104241
[2023-10-18T10:42:41.470+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 77 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1536)
[2023-10-18T10:42:41.502+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:41.602+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:11.692+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [queued]>
[2023-10-18T11:17:11.703+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [queued]>
[2023-10-18T11:17:11.704+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:11.722+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-15 00:00:00+00:00
[2023-10-18T11:17:11.739+0000] {standard_task_runner.py:57} INFO - Started process 348 to run task
[2023-10-18T11:17:11.744+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-15T00:00:00+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp7hpi_q7a']
[2023-10-18T11:17:11.745+0000] {standard_task_runner.py:85} INFO - Job 108: Subtask waiting_for_data
[2023-10-18T11:17:11.810+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:11.895+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-15T00:00:00+00:00'
[2023-10-18T11:17:11.907+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:11.908+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:11.908+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:11.909+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:11.921+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210315T000000, start_date=20231018T111711, end_date=20231018T111711
[2023-10-18T11:17:11.964+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:11.999+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:50.175+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [queued]>
[2023-10-19T12:11:50.186+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [queued]>
[2023-10-19T12:11:50.186+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:50.204+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-15 00:00:00+00:00
[2023-10-19T12:11:50.211+0000] {standard_task_runner.py:57} INFO - Started process 395 to run task
[2023-10-19T12:11:50.214+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-15T00:00:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpg714q7_o']
[2023-10-19T12:11:50.216+0000] {standard_task_runner.py:85} INFO - Job 105: Subtask waiting_for_data
[2023-10-19T12:11:50.270+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-15T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:11:50.360+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-15T00:00:00+00:00'
[2023-10-19T12:11:50.370+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:11:50.371+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:11:50.371+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:11:50.371+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:11:50.381+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210315T000000, start_date=20231019T121150, end_date=20231019T121150
[2023-10-19T12:11:50.439+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:11:50.467+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
